includes/object-cache.php

```diff
@@ -2417,7 +2417,21 @@ public function add_global_groups( $groups ) {
      * @param array $groups  List of groups that are to be ignored.
      */
     public function add_non_persistent_groups( $groups ) {
-        $groups = (array) $groups;
+        $groups          = (array) $groups;
+        $default_groups  = [ 'counts', 'plugins', 'themes' ];
+
+        static $groups_override        = null;
+        static $removed_default_groups = null;
+
+        if ( is_null( $groups_override ) && defined( 'WP_REDIS_IGNORED_GROUPS' ) && is_array( WP_REDIS_IGNORED_GROUPS ) ) {
+            $groups_override        = array_map( [ $this, 'sanitize_key_part' ], WP_REDIS_IGNORED_GROUPS );
```

Comment:
`sanitize_key_part()` is no longer needed here.

Reply:
I copied that from what was done in `__construct()` for `$this->ignored_groups`. Sure you want to remove it? Let me know and I'll make the change.

Reply:
Yes, I removed it in the 2.1.x release.

Reply:
I synced my branch with `develop` and still see it being done in `__construct()` :thinking: :

https://github.com/lots0logs/redis-cache/blob/patch-1/includes/object-cache.php#L490-L492